src/System.IO.Pipes/tests/System.IO.Pipes.Tests.csproj

```diff
@@ -36,6 +36,9 @@
     <Compile Include="PipeTest.Read.cs" />
     <Compile Include="PipeTest.Write.cs" />
     <Compile Include="XunitAssemblyAttributes.cs" />
+    <Compile Include="$(CommonTestPath)\System\Threading\Tasks\TaskTimeoutExtensions.cs">
```

Comment:
Leftover of some experimentation with pipes?

Reply:
Nope, on purpose.  System.IO.Pipes had its own copy of the helper; I removed the copy and it's now using the single copy I modified.

Reply:
Oh missed that the method was removed from there.